### DISCUSSION TITLE
Version 12.1.1.2 def MQTT Host only IP nummer working

### DISCUSSION THREAD

**Discussion by Mark-Muc** [arendst/Tasmota#16599] [Q&A] [answered]
Hello . just found out in Version 12.1.1.2 (dev) only the host name for MQTT Server has to be enterd / will be connected just with IP # of the server.
i.e. may MQTT server hast the name on the network "node-red" oder "Node-RED" .. just built a ne version with Tasmotacompiler (12.1.1.2 ) and try to connect it with MQTT .. only IP # works .. DNS name don't work anymore.
on 12.1.1.1 it works fine .. 
Tested on a shelly 2.5!
Regards
Mark

**Comment by sfromis** [comment, marked as answer]
With 12.1.1.2, just tested with `MqttHost myserver` instead of using an IP. No problem with the DNS name.

**Reply by Mark-Muc** [reply]
OK . . I switched back to 12.1.1.1 because also date and time are not updated .. it was on 1970xxx
these day's I will built a newer version an will have a look!!
Regard 
Mark

**Reply by sfromis** [reply]
Seeing 1970 indicates that it also failed to contact a NTP serer for real time

**Comment by Mark-Muc** [comment]
Hello .. today a made a new version 12.1.1.2 (dev) and same behavior ..
MQTT Host DNS name -> not working
MQTT Host IP# -> working
![image](https://user-images.githubusercontent.com/67410847/191501945-ce9cd030-1f65-45c9-a868-5ee43d89b22e.png)
Regards 
Mark

**Reply by barbudor** [reply]
If DNS is not working it should be a problem in your network configuration

I have no problem using a name for my MqttHost:
```
00:00:00.001 HDW: ESP8266EX
00:00:00.046 CFG: Loaded from flash at FB, Count 11
00:00:00.122 Project tasmota - dev-4119 Version 12.1.1.2(tasmota)-2_7_4_9(2022-09-21T00:01:19)
00:00:03.600 WIF: Connecting to AP1 XXXXXXXXXXX Channel 1 BSSId XX:XX:XX:XX:XX:XX in mode 11n as dev-4119...
00:00:04.755 WIF: Connected
00:00:05.008 HTP: Web server active on dev-4119 with IP address 192.168.168.199
20:11:28.009 MQT: Attempting connection...
20:11:28.053 MQT: Connected
20:11:28.056 MQT: tele/dev-4119/LWT = Online (retained)
20:11:28.058 MQT: cmnd/dev-4119/POWER = 
20:11:28.066 MQT: tele/dev-4119/INFO1 = {"Info1":{"Module":"Generic","Version":"12.1.1.2(tasmota)","FallbackTopic":"cmnd/3C71BF2AD017_fb/","GroupTopic":"cmnd/alldevices/"}}
20:11:28.074 MQT: tele/dev-4119/INFO2 = {"Info2":{"WebServerMode":"Admin","Hostname":"dev-4119","IPAddress":"192.168.168.199"}}
20:11:28.084 MQT: tele/dev-4119/INFO3 = {"Info3":{"RestartReason":"Software/System restart","BootCount":4}}
20:11:32.480 MQT: tele/dev-4119/STATE = {"Time":"2022-09-21T20:11:32+02:00","Uptime":"0T00:00:12","UptimeSec":12,"Heap":28,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":1,"Wifi":{"AP":1,"SSId":"BNETWLAN","BSSId":"88:C3:97:F2:25:EC","Channel":1,"Mode":"11n","RSSI":82,"Signal":-59,"LinkCount":1,"Downtime":"0T00:00:06"}}
20:11:38.258 CMD: mqtthost
20:11:38.264 MQT: stat/dev-4119/MQTTHOST = {"MqttHost":"server"}
20:11:49.107 CMD: ping server
20:11:49.118 MQT: stat/dev-4119/PING = {"Ping":"Done"}
20:11:50.301 MQT: tele/dev-4119/PING = {"Ping":{"server":{"Reachable":true,"IP":"192.168.168.244","Success":1,"Timeout":0,"MinTime":6,"MaxTime":6,"AvgTime":6}}}
```
